operations/deployment/terraform/bitops.after-deploy.d/delete-tf-state-bucket.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [[ $TERRAFORM_DESTROY = true ]] ; then
-  echo "Destroying S3 buket"
+if [[ $TF_STATE_BUCKET_DESTROY = true ]] ; then
```

Comment:
```suggestion
if [[ $TERRAFORM_DESTROY = true ]] && [[ $TF_STATE_BUCKET_DESTROY = true ]]; then 
```

Reply:
should only be able to destroy the bucket if we're destroying everything else, too

Reply:
Resolved
```
[Bitovi] [github-actions-deploy-stackstorm] $ export TF_STATE_BUCKET_DESTROY=true; export TERRAFORM_DESTROY=true; ./operations/deployment/terraform/bitops.after-deploy.d/delete-tf-state-bucket.sh
Destroying S3 buket -->
```